### ISSUE TITLE
After the first successful installation, I cannot log in using the default account and password

### ISSUE THREAD

**Issue by Kirito-F** [openfans-community-offical/Debian-Pi-Aarch64#208]
The installation process is normal, but when the process ends, the login interface I see cannot log in with the default account and password. This is true both on the desktop and SSH.
The account I tried was pi and the password was raspberry.
![ssh](https://user-images.githubusercontent.com/65099034/132161732-e3a1eac1-00a2-4d70-bf39-8760b0987178.jpg)
![desktop](https://user-images.githubusercontent.com/65099034/132161737-0574c225-495e-4fb9-87e2-3ee72b842819.jpg)

**Comment by aiminickwong** [comment]
Any image's version?  And check you img's md5 1st.

**Comment by Kirito-F** [comment]
I used 2021-06-11-OPENFANS-Debian-Buster-Desktop-Aarch64-ext4-v2021-2.0-U6-Release.img.
And I just successfully verified MD5. Then I tried to reinstall it.
I noticed that during the installation process, I can successfully enter the desktop. I tried to click various setting interfaces for a short time, which is normal. But soon the system automatically restarted. As shown in the second picture, the screen was still black after waiting for half an hour. So I tried to power off and restart. After restarting, I entered the login interface and tried the default account and password. The problem remains the same.
![desktop](https://user-images.githubusercontent.com/65099034/132212958-27ba3be2-e01c-41b7-b0c6-e10d9a4aa335.jpg)
![restarted](https://user-images.githubusercontent.com/65099034/132213009-9399833a-3b4e-42ed-8978-6585ca6e87f6.jpg)

**Comment by aiminickwong** [comment]
Would you please provide the size of the SD card and the model of the PI?

**Comment by aiminickwong** [comment]
Do not "tried to click various setting interfaces for a short time" before system auto reboot

**Comment by aiminickwong** [comment]
And  what settings U did?

**Comment by Kirito-F** [comment]
> Would you please provide the size of the SD card and the model of the PI?

The size of the SD card is 64 GB. The model of the PI is 4B Rev 1.2 .

**Comment by Kirito-F** [comment]
> And what settings U did?

I didn't change any settings that time. I just tried to see if the interface can open normally.

**Comment by Kirito-F** [comment]
I reinstalled the system again. There is such a prompt after the first boot.
![prompt](https://user-images.githubusercontent.com/65099034/132330964-dbc98a52-0118-4333-949b-dcd67b5e9364.jpg)
I didn't do anything. Soon after, the system automatically restarted. After such an error, the restart process was interrupted.
![error](https://user-images.githubusercontent.com/65099034/132331022-770c7ea9-fa9e-47a7-8246-03515ed6cfe7.jpg)

**Comment by Kirito-F** [comment]
Finally, I tried to replace another memory card, and the problem was solved, but the previous memory card also passed a simple read-write test. I can guarantee that the previous memory card was purchased through formal channels. I don't know if this is a compatibility problem, but the problem has been solved, and I decided not to go deep into it.
Here are some information:
The previous memory card was jointly signed by banq and JD. It has 64g storage space and the speed standards of V30, A1 and U3. I later tried an ordinary Kingston memory card, which also has 64g storage space, but the speed standard is only C10.
